hr_holiday_notify_employee_manager/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+HR Holiday Notify Employee Manager
+==================================
+
+This module extends the functionality of the Leaves App to make possible
```

Comment:
In any case, maybe you can explicit the difference here with the standard notification system